.expeditor/config.yml

```diff
@@ -7,8 +7,12 @@ slack:
 
 pipelines:
   - docker/build
+  - docker/build_windows
```

Comment:
you need to add a definition to the pipeline. 
https://expeditor.chef.io/docs/getting-started/pipelines/#best-practices--naming-conventions 

```
- docker/build_windows
   definition: .expeditor/build_windows.docker.yml   
```

Reply:
requested changes added, pls review back